ci/matrix.sh

```diff
@@ -55,7 +55,7 @@ elif [ "$BUILD_TARGET" = "win64" ]; then
   export DIRECT_WINE_EXEC_TESTS=true
 elif [ "$BUILD_TARGET" = "linux32" ]; then
   export HOST=i686-pc-linux-gnu
-  export BITCOIN_CONFIG="--enable-zmq --enable-glibc-back-compat --enable-reduce-exports --enable-crash-hooks LDFLAGS=-static-libstdc++"
+  export BITCOIN_CONFIG="--enable-zmq --disable-bip70 --enable-glibc-back-compat --enable-reduce-exports --enable-crash-hooks LDFLAGS=-static-libstdc++"
```

Comment:
Should we update other targets in this file, e.g., `linux64` right below, to include the `--disable-bip70` option too?

Reply:
Having `--disable-bip70` in `linux32` job only follows 15411's logic, should be enough imo.